src/include/esp-idf/bindings.h

```diff
@@ -15,6 +15,11 @@
 #include "esp_ipc.h"
 #include "esp_mac.h"
 
+#include "freertos/task_snapshot.h"
+
+#include "esp_spiram.h"
```

Comment:
Requires the `esp_psram` component on ESP IDF 5+ to be enabled. And for ESP-IDF+ we should ideally include `esp_psram.h`, not `esp_spiram.h`.

Reply:
Good point, I didn't check ESP-IDF 5. Thanks